CRM/Core/BAO/Mapping.php

```diff
@@ -115,7 +115,7 @@ public static function add($params) {
    *   Array of mapping names, keyed by id.
    */
   public static function getMappings($mappingType) {
-    $result = civicrm_api3('Mapping', 'get', array('mapping_type_id' => $mappingType, 'options' => array('limit' => 1, 'sort' => 'name')));
+    $result = civicrm_api3('Mapping', 'get', array('mapping_type_id' => $mappingType, 'options' => array('sort' => 'name')));
```

Comment:
Not a must, but it might be nice to split this very long line into shorter ones while you're changing it, maybe

```
    $options = array('sort' => 'name');
    $params = array('mapping_type_id' => $mappingType, 'options' => $options);
    $result = civicrm_api3('Mapping', 'get', $params);
```

Another suggestion would be to use `array_column($result['values'], 'name')` instead of the for loop, but that might be out of the scope of this.